readme.md

```diff
@@ -278,12 +278,18 @@ Type: [`ITokenizer`](https://github.com/Borewit/strtok3#tokenizer)
 
 A file source implementing the [tokenizer interface](https://github.com/Borewit/strtok3#tokenizer).
 
-### FileType.stream(readableStream)
+### FileType.stream(readableStream, sampleSize)
```

Comment:
This needs to be properly documented as an options-object. Example: https://github.com/sindresorhus/boxen#options